### DISCUSSION TITLE
x6-react-shape v2.0.6 not support react:17.x

### DISCUSSION THREAD

**Discussion by ederzz** [antvis/X6#3187] [Q&A] [answered]
In this [pr](https://github.com/antvis/X6/pull/2968)，maintainer has done some work for support `react@17.x`.When i use the package `@antv/x6-react-shape@2.0.6`，i also got the error: 

```
Module not found: Error: Can't resolve 'react-dom/client' in '...\node_modules\@antv\x6-react-shape\es'
```

Is there a way to use `x6-react-shape@2.x` in `react@17.x`？

**Comment by ederzz** [comment, marked as answer]
The correct version is : 2.0.8+ ~ 2.1.x

**Reply by NewByVector** [reply]
https://www.yuque.com/antv/x6/tox1ukbz5cw57qfy
